<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSudoku">
    <property name="title">
     <string>Sudoku</string>
    </property>
    <widget class="QMenu" name="menuDifficulties">
     <property name="title">
      <string>Difficulties</string>
     </property>
     <addaction name="actionEasy"/>
     <addaction name="actionNormal"/>
     <addaction name="actionHard"/>
    </widget>
    <addaction name="generateAction"/>
    <addaction name="clearAction"/>
    <addaction name="menuDifficulties"/>
    <addaction name="separator"/>
    <addaction name="toggleSolutionAction"/>
   </widget>
   <addaction name="menuSudoku"/>
  </widget>
  <action name="generateAction">
   <property name="text">
    <string>New Board</string>
   </property>
  </action>
  <action name="clearAction">
   <property name="text">
    <string>Clear Board</string>
   </property>
  </action>
  <action name="toggleSolutionAction">
   <property name="text">
    <string>Show Solution</string>
   </property>
  </action>
  <action name="actionEasy">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Easy</string>
   </property>
  </action>
  <action name="actionNormal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Normal</string>
   </property>
  </action>
  <action name="actionHard">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hard</string>
   </property>
  </action>
  <action name="actionSuper_Hard">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Super Hard</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
